### PR TITLE
Bug fix - Artifactory uploader log message & Usage Doc update

### PR DESCRIPTION
**Pull request by adityanarayanp**
**Bug Fix**
- Fixing the artifactory uploader log message to show "copy" instead of "move" for development packages - as they are copied from the remote repo to the siparty-devdev-* repo.
- Updating the artifactory uploader KPI to "copy" instead of "move" for dev packages.

**Updates**
- Updating the usage document to mention that the SEPP tool has been integrated with the artifactory uploader.
- Adding the JFrog artifactory error response message to the log file.

### CODE REVIEW

File: doc/UsageDoc/CA_UsageDocument.md [hunk 1]
```diff
@@ -52,6 +52,12 @@ The Continuous Clearing Tool helps the Project Manager/Developer to automate the
 
 Continuous Clearing Tool reduces the effort in creating components in SW360 and identifying the matching source codes from the public repository. Tool eliminates the manual error while creating component and identifying correct version of source code from public repository. Continuous Clearing Tool harmonize the creation of 3P components in SW360 by filling necessary information.
 
+The Continuous Clearing Tool incorporates SEPP tool functionalities, seamlessly integrated into the [Artifactory uploader](../usagedocimg/artifactoryuploader.PNG).
```
[the remark this answers is not in the document]
Comment (karthika-g) [reply doc/UsageDoc/CA_UsageDocument.md:55]: Add few lines in Readme.md as well

[the remark this answers is not in the document]
Comment (adityanarayanp) [reply doc/UsageDoc/CA_UsageDocument.md:57]: have updated it from the Wiki

[the remark this answers is not in the document]
Comment (karthika-g) [reply doc/UsageDoc/CA_UsageDocument.md:57]: and readme

[the remark this answers is not in the document]
Comment (adityanarayanp) [reply doc/UsageDoc/CA_UsageDocument.md:57]: Updated the readMe

**Review by karthika-g — APPROVED** [review]
approved

**Pull request merged**